index.css

```diff
@@ -0,0 +1,222 @@
+::after, ::before {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  line-height: 1.5;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
+  background-color: white;
+  font-size: 16px;
+  color: #212529;
+}
+
+/* ===  Navigation Section ===  */
+.NavContainer {
+  display: flex;
+  margin: 0 0 16px;
+  padding: 16px 24px;
+  border-bottom: 1px solid #dee2e6;
+  box-shadow: 2px 2px 1px rgba(0,0,0,.075);
+}
+
+.Logo {
+  align-self: center;
+}
+
+.logoitem {
+  font-size: 20px;
+  margin: 0;
+  text-decoration: none;
+  font-weight: 400;
+}
+.NavList {
+  flex-grow: 1;
+  align-self: center;
+
+}
+.NavListitems {
+  list-style-type: none;
+  margin: 0 16px 0 0;
```

Comment:
if u want to change only `margin-right` this property, but if you really want to set right and reset all others - it is ok